src/GroupInterface.php

```diff
@@ -46,4 +46,6 @@ public function host(string $host): self;
     public function namePrefix(string $namePrefix): self;
 
     public function routes(...$routes): self;
+
+    public function withAutoOptions(...$middlewares): self;
```

Comment:
Need phpdoc.

Reply:
`withAutoOptionsMiddlewares()`?

Reply:
No, this method is not only for middlewares, it appends OPTIONS routes.

Reply:
OK.